bin/test.js

```diff
@@ -126,44 +138,34 @@ export class WTRConfig {
 	}
 
 	#getPattern(type) {
-		const pattern = this.#cliArgs.files || this.pattern(type);
+		const pattern = [].concat(this.#cliArgs.files || this.pattern(type));
 
 		// replace filename wildcards with all filter strings
 		// e.g. If filter is ['button', 'list'], pattern './test/*.test.*' becomes:
 		// [ './test/+(*button*.test.*|*.test.*button*)', './test/+(*list*.test.*|*.test.*list*)' ]
 		if (this.#cliArgs.filter) {
 			return this.#cliArgs.filter.map(filterStr => {
 				// replace everything after the last forward slash
-				return pattern.replace(/[^/]*$/, fileGlob => {
+				return pattern.map(p => p.replace(/[^/]*$/, fileGlob => {
 					// create a new glob for each wildcard
 					const fileGlobs = Array.from(fileGlob.matchAll(/(?<!\*)\*(?!\*)/g)).map(({ index }) => {
 						const arr = fileGlob.split('');
-						arr.splice(index, 1, `*${filterStr}*`);
+						arr.splice(index, 1, filterStr);
 						return arr.join('');
 					});
-					return `+(${fileGlobs.join('|')})`;
-				});
-			});
+					return `+(${fileGlobs.join('|') || fileGlob})`;
+				}));
+			}).flat();
 		}
 		return pattern;
 	}
 
 	create({
 		pattern = DEFAULT_PATTERN,
-		vdiff = DEFAULT_VDIFF,
 		timeout,
 		...passthroughConfig
 	} = {}) {
-
-		const { files, filter, golden, grep, group, manual, playwright, watch } = this.#cliArgs;
-
-		if (!group || group === 'default') {
-			if (playwright) {
-				console.warn('Warning: reducedMotion disabled. Use the unit group to enable reducedMotion.');
-			} else {
-				console.warn('Warning: Running with puppeteer, reducedMotion disabled. Use the unit group to use playwright with reducedMotion enabled');
-			}
-		}
```

Comment:
No longer necessary since we are disabled `default` and group and defaulting to `unit` (and forcing all `browsers` options to playwright browsers).